cmd/pebble-client/main.go

```diff
@@ -33,7 +33,7 @@ func userAgent() string {
 
 type client struct {
 	server    *url.URL
-	directory map[string]string
+	directory map[string]interface{}
```

Comment:
It was necessary after the addition of the `meta` entry. At that point the directory isn't strictly strings. `pebble-client` was panic'ing after the recent ToS changes without it.